README.md

```diff
@@ -9,20 +9,17 @@
   </a>
 </div>
 
-[Temporal](https://temporal.io) is a microservice orchestration platform that enables developers to build scalable applications without sacrificing productivity or reliability. Temporal Server executes units of application logic—Workflows—in a resilient manner that automatically handles intermittent failures and retries failed operations.
+[Temporal](https://temporal.io) is a distributed, scalable, durable, and highly available orchestration engine used to execute asynchronous, long-running business logic in a scalable and resilient way.
 
-Temporal is a mature technology—it's a fork of Uber's Cadence. Temporal is being developed by Temporal Technologies, a startup by the creators of Cadence.
+"Temporal TypeScript SDK" is the framework for authoring workflows and activities using either the TypeScript or JavaScript programming languages.
 
-## Documentation
-
-The documentation is divided into several sections:
+For getting started and general documentation, please refer to the following resources:
```

Comment:
Had to read the "For getting started" part several times before I got to parse the correct meaning of this sentence. Not sure how to best fix, but you can consider this:

```suggestion
For "Getting started instructions" and General documentation, please refer to the following resources:
```

Reply:
Let me think about this one.